zebra-scan/src/tests.rs

```diff
@@ -183,36 +184,37 @@ async fn scanning_zecpages_from_populated_zebra_state() -> Result<()> {
     .unwrap();
 
     let account = AccountId::from(1);
-    let nf = Nullifier([7; 32]);
 
     // Build a vector of viewing keys `vks` to scan for.
     let fvk = efvk.fvk;
     let ivk = fvk.vk.ivk();
     let vks: Vec<(&AccountId, &SaplingIvk)> = vec![(&account, &ivk)];
 
     let network = zebra_chain::parameters::Network::default();
+    let state_config = Default::default();
 
-    // Create a continuous chain of mainnet blocks from genesis
-    let blocks: Vec<Arc<Block>> = zebra_test::vectors::CONTINUOUS_MAINNET_BLOCKS
-        .iter()
-        .map(|(_height, block_bytes)| block_bytes.zcash_deserialize_into().unwrap())
-        .collect();
-
-    // Create a populated state service.
     let (_state_service, read_only_state_service, latest_chain_tip, _chain_tip_change) =
-        zebra_state::populated_state(blocks.clone(), network).await;
-
+        zebra_state::spawn_init(state_config, network, zebra_chain::block::Height::MAX, 3000)
```

Comment:
How many blocks will this scan ? I think 0 blocks if the state is empty (which is always the case when running this test).

Reply:
Looks good now, thanks!